graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/KotlinCodeGenTest.kt

```diff
@@ -2780,23 +2780,37 @@ class KotlinCodeGenTest {
     }
 
     @Test
-    fun deprecateAnnotationWithNoMesssage() {
+    fun deprecateAnnotationWithNoReason() {
```

Comment:
Nit: since we're changing it anyway, can we change it to be correct (there _is_ a reason in this test case).

Reply:
I will make it correct by removing the reason attribute so that its missing from both deprecated directives.  And for consistency, I will also update the corresponding test in CodeGenTest to test for the same scenario.